macros/quote_something.sql

```diff
@@ -0,0 +1,22 @@
+{%- macro quote(thing) -%}
+
+{{ adapter.dispatch('quote', 'fivetran_utils') (thing) }}
```

Comment:
Similar question regarding the dispatch. Is there a reason this does not have the `return` statement?

Reply:
added `return`